Shorten fragment reuse guide (#4033)
viamrobotics/docs: docs/manage/fleet/reuse-configuration.md

{{< table >}}
{{% tablestep link="/operate/get-started/setup/" %}}
**1. In the [Viam app](https://app.viam.com), create a machine** in any {{< glossary_tooltip term_id="location" text="location" >}}.

{{% /tablestep %}}
{{% tablestep link="/operate/get-started/setup/" %}}

> **2. Use the CONFIGURE tab to configure all resources** you want to use on all your machines.

Reviewer: ```suggestion
**2. Use the CONFIGURE tab to configure all the resources** you want to use on your machines.
```
Otherwise sounds almost like you're configuring them on each of your machines at this step, and reads oddly

All available resources except [triggers](/data-ai/data/advanced/alert-data/) are supported in fragments.

> This means you can also use fragments inside a fragment.

Reviewer: ```suggestion
You can even configure other fragments inside a fragment.
```

{{% /tablestep %}}
{{% tablestep %}}
**3. Switch to JSON and copy the raw JSON configuration object**.

> {{<imgproc src="/how-tos/one-to-many/raw-json.png" resize="700x" class="shadow fill" style="width: 400px" declaredimensions=true alt="JSON subtab of the CONFIGURE tab">}}

Reviewer: [opt] Now with the red box, I worry folks might lose the focus on actually copying the JSON text itself. Consider taking a screenshot where the JSON config is selected, ready to copy

> 

Reviewer: [out of diff] Do we still need to say "Fragment overwrites are currently not supported for modifying triggers" if triggers themselves aren't supported in fragments?
